lib/topology/injection.py

```diff
@@ -155,7 +155,13 @@ def subfolders(search_path):
                         result[filename][node] = {}
 
                     for attribute, value in modifier['attributes'].items():
-                        result[filename][node][attribute] = value
+                        if isinstance(value, dict):
```

Comment:
This changes the way attribute injection files are used by allowing dictionaries as possible values of the attributes in a modifier. If this were to be added, it would be a new feature that would need to be committed along with describing documentation. May I ask what high-level purpose are you trying to achieve here? Are you trying to add dictionaries as node attributes?